doxygen/examples/H5I_examples.c

```diff
@@ -133,7 +133,7 @@ fail_dcpl:;
         hid_t      obj_id;
 
         // register a new ID type
-        if ((type = H5Iregister_type(128, 1024, &free_func)) < 0) {
+        if ((type = H5Iregister_type2(1024, &free_func)) < 0) {
```

Comment:
I don't think this change should be going into a minor release

Reply:
Will fix